ahoy.go

```diff
@@ -100,38 +100,39 @@ func getConfig(sourcefile string) (Config, error) {
 
 func getSubCommands(path string) []cli.Command {
 	subCommands := []cli.Command{}
-	if path != "" {
-		includes := strings.Split(path, "\n")
-		commands := map[string]cli.Command{}
-		for _, include := range includes {
-			if len(include) == 0 {
-				continue
-			}
-			includeSource := include
-			if include[0] != "/"[0] || include[0] != "~"[0] {
-				includeSource = filepath.Join(sourcedir, include)
-			}
-			if _, err := os.Stat(includeSource); err != nil {
-				//Skipping files that cannot be loaded allows us to separate
-				//subcommands into public and private.
-				continue
-			}
-			config, _ := getConfig(includeSource)
-			includeCommands := getCommands(config)
-			for _, command := range includeCommands {
-				commands[command.Name] = command
-			}
+	if path == "" {
+		return subCommands
+	}
+	includes := strings.Split(path, "\n")
+	commands := map[string]cli.Command{}
+	for _, include := range includes {
+		if len(include) == 0 {
+			continue
 		}
-
-		var names []string
-		for k := range commands {
-			names = append(names, k)
+		includeSource := include
+		if include[0] != "/"[0] || include[0] != "~"[0] {
+			includeSource = filepath.Join(sourcedir, include)
+		}
+		if _, err := os.Stat(includeSource); err != nil {
+			//Skipping files that cannot be loaded allows us to separate
```

Comment:
Thought about this overnight, I don't actually like the idea of adding anything to the path.  This will just add unnecessary complexity.  I think a warning  (preferably logged) should suffice.  Which brings me to a pet peeve I have with the current ahoy, the whole thing breaks when it can't find one file... not just commands in that file, the whole thing.